Source/Extensions/StringExtensions.swift

```diff
@@ -144,6 +144,19 @@ public extension String {
 	public var isNumeric: Bool {
 		return  !hasLetters && hasNumbers
 	}
+
+    /// SwifterSwift: Check if string contains only unique characters.
+    public var hasUniqueCharacters: Bool {
```

Comment:
@FrankKair thank you for the extension 👍 Is there any chance we can turn this into a method? Users normally associate properties with values that are quick to compute or are stored internally. This method has O(n) complexity and could be a heavy operation on a long String.